<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="Balaji-Note" name="com.esri.test.auto.testcases.TC04_ImageSubPortalVerify" tests="1" failures="0" timestamp="1 Sep 2016 20:28:35 GMT" time="0.004" errors="1">
  <testcase name="imageSubmissionPortalVerify" time="0.004" classname="com.esri.test.auto.testcases.TC04_ImageSubPortalVerify">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.esri.test.auto.wrappers.GenericWrappers.VerifyTitle(GenericWrappers.java:260)
	at com.esri.test.auto.pages.ImageSubPortalVerify.<init>(ImageSubPortalVerify.java:13)
	at com.esri.test.auto.testcases.TC04_ImageSubPortalVerify.imageSubmissionPortalVerify(TC04_ImageSubPortalVerify.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
=======
<testsuite hostname="IMPC3478" name="com.esri.test.auto.testcases.TC04_ImageSubPortalVerify" tests="1" failures="0" timestamp="6 Sep 2016 16:50:16 GMT" time="0.002" errors="1">
  <testcase name="imageSubmissionPortalVerify" time="0.002" classname="com.esri.test.auto.testcases.TC04_ImageSubPortalVerify">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.testng.internal.ArrayIterator.hasNext(MethodHelper.java:350)
	at org.testng.internal.Parameters$2.hasNext(Parameters.java:448)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1117)
>>>>>>> branch 'master' of https://github.com/Esribalajih/ESRI-SEL-Auto
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- imageSubmissionPortalVerify -->
</testsuite> <!-- com.esri.test.auto.testcases.TC04_ImageSubPortalVerify -->
